File: web-development/level-1/helsinki-part-1-2/Java/part1/week4/week04-ex80-RollingTheDice/Dice.java
//dice class
import java.util.Random;

public class Dice {
  private Random random;
  private int numberOfSides;

  //constructor
  public Dice(int numberOfSides) {
    this.numberOfSides = numberOfSides;
    random = new Random();
  }

  public int roll() {
    return random.nextInt(6) + 1;
  }

  
}

File: compsci/programming/helsinki-moocs/python/ex80/Dice.py
from random import randint

class Dice:

    #constructor
    def __init__(self, number_of_sides):
        self.side_number = number_of_sides
        self.random_num = 0

    #behaviors

    #rolls the Dice
    def roll(self):
        self.random_num = randint(1, self.side_number)
        return self.random_num
